core/src/main/java/com/novoda/noplayer/drm/provision/HttpPoster.java

```diff
@@ -6,7 +6,7 @@
 import java.net.HttpURLConnection;
 import java.net.URL;
 
-class HttpPoster {
+public class HttpPoster {
```

Comment:
I would rather keep this `package` and in case make a `newInstance`  on the `ProvisioningExecutor`. I think that's one of the little benefits, on my opinion, of the `newInstance` and therefore I don't see justification from removing it from `ProvisioningExecutor`